mappings/net/minecraft/client/render/entity/EntityRenderDispatcher.mapping

```diff
@@ -15,45 +15,55 @@ CLASS net/minecraft/class_898 net/minecraft/client/render/entity/EntityRenderDis
 	METHOD <init> (Lnet/minecraft/class_1060;Lnet/minecraft/class_918;Lnet/minecraft/class_3296;Lnet/minecraft/class_327;Lnet/minecraft/class_315;)V
 		ARG 1 textureManager
 	METHOD method_17145 register (Lnet/minecraft/class_1299;Lnet/minecraft/class_897;)V
-	METHOD method_23161 fireVertex (Lnet/minecraft/class_4587$class_4665;Lnet/minecraft/class_4588;FFFFF)V
-		ARG 0 matrix
-		ARG 1 vertexConsumer
+	METHOD method_23161 drawFireVertex (Lnet/minecraft/class_4587$class_4665;Lnet/minecraft/class_4588;FFFFF)V
+		ARG 0 entry
+		ARG 1 vertices
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
 		ARG 5 u
 		ARG 6 v
-	METHOD method_23162 shadowVertex (Lnet/minecraft/class_4587$class_4665;Lnet/minecraft/class_4588;FFFFFF)V
-		ARG 0 matrix
-		ARG 1 vertexConsumer
+	METHOD method_23162 drawShadowVertex (Lnet/minecraft/class_4587$class_4665;Lnet/minecraft/class_4588;FFFFFF)V
+		ARG 0 entry
+		ARG 1 vertices
 		ARG 2 alpha
 		ARG 3 x
 		ARG 4 y
 		ARG 5 z
 		ARG 6 u
 		ARG 7 v
 	METHOD method_23163 renderShadowPart (Lnet/minecraft/class_4587$class_4665;Lnet/minecraft/class_4588;Lnet/minecraft/class_4538;Lnet/minecraft/class_2338;DDDFF)V
-		ARG 0 matrix
-		ARG 1 vertexConsumer
+		ARG 0 entry
+		ARG 1 vertices
 		ARG 2 world
 		ARG 3 pos
 		ARG 4 x
 		ARG 6 y
 		ARG 8 z
-		ARG 10 size
-		ARG 11 darkness
-	METHOD method_23165 renderFire (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;Lnet/minecraft/class_1297;)V
+		ARG 10 radius
+		ARG 11 opacity
+	METHOD method_23164 drawBox (Lnet/minecraft/class_4587;Lnet/minecraft/class_4588;Lnet/minecraft/class_1297;FFF)V
 		ARG 1 matrix
+		ARG 2 vertices
```

Comment:
`vertexConsumer`.

Reply:
If it's a VertexConsumer naming it `vertices` is intended.